repos/fdbt-site/tests/pages/api/inboundMatching.test.ts

```diff
@@ -61,7 +61,7 @@ describe('Inbound Matching API', () => {
 
     it('correctly generates matching error info, updates the INBOUND_MATCHING_ATTRIBUTE and then redirects to inboundMatching page when there are unassigned fare stages', () => {
         const mockMatchingError: MatchingWithErrors = {
-            error: true,
+            warning: true,
```

Comment:
Should there be a test for error here as well? The old test was still valid I think?

Reply:
having trouble getting this to work :/ 

I'm able to set the expected data to error but unsure what I need to feed in the get the actual to output an error